indico/modules/events/editing/client/js/management/editable_type/EditableTypeDashboard.jsx

```diff
@@ -75,6 +80,12 @@ export default function EditableTypeDashboard() {
     slides: Translate.string('Assign an editor to slides'),
   }[type];
 
+  const {data} = useIndicoAxios({
+    url: contribsWithNoEditablesURL({event_id: eventId, type}),
+    method: 'POST',
+  });
+  const numEditables = (data && data.no_contribs) || 0;
```

Comment:
This could be simply `data.count` and maybe `numContribs` instead?

Reply:
yes, absolutely. no need to POST here - it's neither modifying something nor has it a large payload (e.g. list of selected ids)...

Reply:
Optional chaining could also simplify this:
```suggestion
  const numEditables = data?.no_contribs || 0;
```